tesseract_ros/tesseract_examples/tesseract_ros_examples/CMakeLists.txt

```diff
@@ -73,6 +73,26 @@ target_include_directories(${PROJECT_NAME}_glass_up_right_example_node PRIVATE
 target_include_directories(${PROJECT_NAME}_glass_up_right_example_node SYSTEM PRIVATE
     ${catkin_INCLUDE_DIRS})
 
+add_library(${PROJECT_NAME}_glass_up_right_ompl_example src/glass_up_right_ompl_example.cpp)
```

Comment:
Perhaps this is as good a time as any to address the fact that upright is a single word?

Reply:
Updated